pom.xml

```diff
@@ -3,14 +3,14 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.iopipe</groupId>
   <artifactId>iopipe</artifactId>
-  <version>1.10.0-SNAPSHOT</version>
+  <version>${revision}</version>
```

Comment:
This doesn't have -SNAPSHOT, is that ok?

Reply:
For this, this does not have snapshot because this is supposed to be the release version. Although thinking about it, the `revision` variable could just be set to `-SNAPSHOT` so it is not treated as a true release version. Going to do that.